smartconnect/er_sync_utils.py

```diff
@@ -156,7 +156,13 @@ def get_leaf_options_with_config(options, optionsConfig = None):
     return leaf_options
 
 
-def build_schema_and_form_definition(*, attributes: List[Attribute], leaf_attributes: List[CategoryAttribute], is_multiple: bool, attributeConfigs = None):
+def build_schema_and_form_definition(*args, attributes: List[Attribute] = None,
+                                     leaf_attributes: List[CategoryAttribute] = None,
+                                     is_multiple: bool = False,
+                                     attributeConfigs = None):
+
+    assert not args, "This function takes keyword arguments only"
```

Comment:
just a small question, do you do this `*args` and the `assert` instead of `*` to keep backward compatibility with Python 2?

Reply:
Thanks @marianobrc and sorry for the slow reply. 

Old habits die hard. I'm glad I won't have to do that any more. Thanks!